source/ed/connectors/gtfs_parser.cpp

```diff
@@ -388,85 +388,113 @@ bool StopsGtfsHandler::parse_common_data(const csv_row& row, T* stop) {
     return true;
 }
 
-StopsGtfsHandler::stop_point_and_area StopsGtfsHandler::handle_line(Data& data, const csv_row& row, bool) {
+nm::StopArea* StopsGtfsHandler::build_stop_area(Data& data, const csv_row& row) {
+    nm::StopArea* sa = new nm::StopArea();
+    if (!parse_common_data(row, sa)) {
+        delete sa;  // don't forget to free the data
+        return nullptr;
+    }
+
+    if (has_col(wheelchair_c, row) && row[wheelchair_c] == "1") {
+        sa->set_property(navitia::type::hasProperties::WHEELCHAIR_BOARDING);
+    }
+    gtfs_data.stop_area_map[sa->uri] = sa;
+    data.stop_areas.push_back(sa);
+
+    if (has_col(timezone_c, row)) {
+        auto tz_name = row[timezone_c];
+        sa->time_zone_with_name = gtfs_data.tz.get_tz(tz_name);
+    }
+    if (!sa->time_zone_with_name.second) {
+        // if no timezone has been found, we set the default one
+        sa->time_zone_with_name = {data.tz_wrapper.tz_name, data.tz_wrapper.boost_timezone};
+    }
+    if (has_col(desc_c, row)) {
+        add_gtfs_comment(gtfs_data, data, sa, row[desc_c]);
+    }
+    return sa;
+}
+
+nm::StopPoint* StopsGtfsHandler::build_stop_point(Data& data, const csv_row& row) {
+    nm::StopPoint* sp = new nm::StopPoint();
+    if (!parse_common_data(row, sp)) {
+        delete sp;
+        return nullptr;
+    }
+
+    if (has_col(zone_c, row) && row[zone_c] != "") {
+        sp->fare_zone = row[zone_c];
+    }
+
+    if (has_col(wheelchair_c, row)) {
+        if (row[wheelchair_c] == "0") {
+            wheelchair_heritance.push_back(sp);
+        } else if (row[wheelchair_c] == "1") {
+            sp->set_property(navitia::type::hasProperties::WHEELCHAIR_BOARDING);
+        }
+    }
+    if (has_col(platform_c, row)) {
+        sp->platform_code = row[platform_c];
+    }
+    gtfs_data.stop_map[sp->uri] = sp;
+    data.stop_points.push_back(sp);
+    if (has_col(parent_c, row) && row[parent_c] != "") {  // we save the reference to the stop area
+        auto it = gtfs_data.sa_spmap.find(row[parent_c]);
+        if (it == gtfs_data.sa_spmap.end()) {
+            it = gtfs_data.sa_spmap.insert(std::make_pair(row[parent_c], GtfsData::vector_sp())).first;
+        }
+        it->second.push_back(sp);
+    }
+
+    if (has_col(desc_c, row)) {
+        add_gtfs_comment(gtfs_data, data, sp, row[desc_c]);
+    }
+
+    // we save the tz in case the stop point is later promoted to stop area
+    if (has_col(timezone_c, row) && !row[timezone_c].empty()) {
+        gtfs_data.tz.stop_point_tz[sp] = row[timezone_c];
+    }
+    return sp;
+}
+
+bool StopsGtfsHandler::check_duplicate(const csv_row& row) {
```

Comment:
minor clearer to me:
```suggestion
bool StopsGtfsHandler::is_duplicate(const csv_row& row) {
```